doc/tensor.md

```diff
@@ -1327,36 +1331,66 @@ x:indexFill(2,torch.LongTensor{4,2},-10)
 ```
 
 <a name="torch.Tensor.gather"/>
-### [Tensor] gather(src, dim, index) ###
+### [Tensor] gather(dim, index) ###
```

Comment:
I think `src` *should* be missing, to be consistent with the rest of the docs which present the functions, on the whole, as method calls. (As it is the docs aren't completely consistent in their style, but not so much as to be confusing. But I did find the particular way that `gather`'s signature was presented was confusing hence the suggestion to change it.)